accepted/2024/vulnerabilities-in-packages.config-restore.md

```diff
@@ -0,0 +1,93 @@
+# ***Vulnerabilities in packages.config restore***
+
+- [Nikolche Kolev](https://github.com/nkolev92)
+- GitHub Issue <https://github.com/NuGet/Home/issues/12307>
+
+## Summary
+
+This is a continuation of the [vulnerabilities in PackageReference work](../2022/vulnerabilities-in-restore.md), extending matching functionality for packages.config restore.
+
+## Motivation
+
+See [vulnerabilities in PackageReference restore motivation](../2022/vulnerabilities-in-restore.md#motivation).
+
+## Explanation
+
+### Functional explanation
+
+Packages.config restore combines all packages regardless of project. Instead of raising warnings and errors on a per project level, at restore time, they're raised on a per package level.
+
+The configuration knobs for packages.config restore auditing functionality will be NuGet.config based.
+In particular, there will be 2 new configuration keys:
+
+| Key | Acceptable values | Description | Default |
+|-----|-------------------|-------------|---------|
+| auditForPackagesConfig | enable, disable | Enables or disables NuGet Audit for packages config projects | If not specified, the default will be `enable` |
+| auditLevelForPackagesConfig | Critical, high, moderate, low | Configures the default audit level for NuGet audit for packages config projects |  If not specified, the default will be `low` |
+
+The audit functionality for packages.config restore will be enabled by default.
+To disable it, one can specify a property in the config section of the configuration file.
```

Comment:
This might get confusing between having this and also a MSBuild property.

Is the rationale here that this would be used for legacy project system?

Reply:
I detail that in https://github.com/NuGet/Home/blob/dev-nkolev92-vulnerabilityCheckingInPCRestore/accepted/2024/vulnerabilities-in-packages.config-restore.md#msbuild-properties-vs-nugetconfig-for-packagesconfig-restore-configuration . 

We have places where we have duplicate configurations, one via nuget.config and another one via property. The property tends to override the nuget.config value. 

The reasoning here is that in packages.config world, for those customers, nuget.config makes more sense.